CTFd/themes/core/templates/welcome.html

```diff
@@ -0,0 +1,26 @@
+{% extends "base.html" %}
+
+{% block stylesheets %}
+{% endblock %}
+
+{% block content %}
+
+<div class="container">
+	<div class="row">
+		<div class="col-md-6 offset-md-3">
+            <br /><br />
+            <h3 class="text-center">Welcome {{ name }} in {{ get_ctf_name() }}!</h3>
```

Comment:
This will look more fluid from the language point of view: `{{ name }}, welcome in {{ get_ctf_name() }}!`